docs/reference/elasticsearch/mapping-reference/geo-point.md

```diff
@@ -236,7 +238,10 @@ PUT idx
   },
   "mappings": {
     "properties": {
-      "point": { "type": "geo_point" }
+      "point": {
+        "type": "geo_point",
+        "synthetic_source_keep": "arrays"
```

Comment:
Do you want to flip the points here to emphasize that order is different? Right now it happens to be a correct order.